tests/main/prepare-image-grub-core18/task.yaml

```diff
@@ -0,0 +1,81 @@
+summary: Check that prepare-image works for grub-systems
+
+systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*, -arch-*]
+
+backends: [-autopkgtest]
+
+# TODO: use the real stores with proper assertions fully as well once possible
+environment:
+    ROOT: /tmp/root
+    IMAGE: /tmp/root/image
+    GADGET: /tmp/root/gadget
+    STORE_DIR: $(pwd)/fake-store-blobdir
+    STORE_ADDR: localhost:11028
+    UBUNTU_IMAGE_SKIP_COPY_UNVERIFIED_SNAPS: 1
```

Comment:
Hmm?

Reply:
That's probably some magical switch to `ubuntu-image`.

Reply:
We don't need this for this particular test, I removed it.